### DISCUSSION TITLE
Oferta Azure AD Solution Architect, zdalnie

### DISCUSSION THREAD

**Discussion by AdrianRITS** [AzureCommunityPL/group#128] [Oferty pracy]
Dzień dobry, 

dla mojego klienta- duża firma z branży FMCG, poszukuję Azure AD Solition Architecta. Zatrudnienie na kontrakcie B2B, stawka do około 1100 MD + VAT. Projekt jest stały. Praca w języku angielskim. Poniżej zamieszczam pełny opis. Oosoby zainteresowane proszę o kontakt mailowy
a.gradzki@rits.center

Role Purpose: 
The  for execution of a proof-of-concept exercise of using Azure AD as Identity Provider (IdP), assisting with creation of Business Case for migrating from OKTA and design, build and migration from OKTA.

Scope and Accountabilities:
•	Document IdP requirements
o	Include details of applications current using OKTA
•	Understand technical capabilities of OKTA vs Azure AD and create gap analysis
•	Define scope and success criteria of POC
•	Plan, design, build and implement POC, testing and measuring against agreed success criteria
•	Build Business Case for migrating from OKTA to Azure AD (inc Commercials)
•	Define migration plan
•	Plan, design, build and implement Azure AD for 
•	Collaborates with colleagues, customers, vendors, and other parties
•	Adheres to (and defines) architecture and cyber standards
•	and situations, including those which may be unfamiliar, in order to address critical issues.

Key Challenges:
•	has a complex landscape that reflects the rapid growth and the global nature of the business. This needs to be streamlined at the same time as driving large scale IS projects forward.  This role will need to quickly establish an in depth understanding of the current landscape and the road map to achieve the desired future state.

Skills & Experience required:
•	Strong vendor management skills
•	Strong organizational skills
•	Strong technical documentation creation and review skills
•	Experience of outsourced offshore support environments
•	Used to work in remotely and in virtual teams with mix of off-shore vendors
•	ITIL best practices
•	Previous experience of GxP environments would be an advantage but not required
•	Broad and proven experience with Azure AD Identity Management in Hybrid environments (AAD Connect)
•	Performed successful implementation of Azure AD MFA in Hybrid environment (ideally in complex environment with many other cloud apps integrated). This includes following elements:
o	Planning
o	Design
o	Build / Implementation
o	Release to Production
•	Experience with design and implementation of Microsoft Single-SignOn
•	Experience with using OKTA as an Identify Provider an advantaged, but not essential
•	Experience in automating user workloads (license assignment, identity management, etc.) using MS toolset.
•	Good PowerShell skills

Personal Attributes: (The personal attributes needed to be successful in the role)
•	Strategist - Demonstrated ability to make significant decisions and recommendations on complex issues to achieve long-term results and strategic objectives.
•	Ability to be flexible and work analytically in a problem solving environment. 
•	Ability to work successfully with minimal supervision in an atmosphere of multiple requirements, shifting priorities, and deadline pressures.
•	Strong interpersonal, communication skills, written, verbal and presentation 
•	Ability to show excellent attention to detail
•	Leadership, organizational & multi-tasking skills
•	Self Confidence
•	Integrity
•	Creative Thinker
•	Ability to work through ambiguity and push for decisions
•	Strong customer focus and management
•	Has a team mentality

**Comment by mgrabarz** [comment]
Ciekawe, ale to chyba robota dla doświadczonego konsultanta.
